packages/meditrak-server/src/apiV2/import/importDataSources/createDataSources.js

```diff
@@ -22,9 +22,9 @@ export async function createDataSources(transactingModels, dataSources) {
 
     const exsitingDataSource = await transactingModels.dataSource.find({ code });
     if (exsitingDataSource.length > 0) {
-      throw new ImportValidationError(`Data source code '${code}' already exists`, excelRowNumber);
+      await transactingModels.dataSource.update({ code }, dataSource);
+    } else {
+      await transactingModels.dataSource.create(dataSource);
     }
```

Comment:
Surprisingly found that code in data_source table is not unique. So we can't use .createOrUpdate(). I believe we can resolve this by Ethan's work on splitting data source table.